liveblog/liveblog
# [LBSD-2767] - Create the static UI to render the polls in Default SEO Theme

### Purpose

This PR implements the user interface for rendering polls in a static manner in the Default SEO theme. The goal is to provide a visual representations of polls, when both opened and closed, to enhance user engagement and interaction

### What has changed

- Added `template-poll.html` to render the polls
- Added `template-poll-utils.html` with helper function to calculate poll votes and time left in order to change poll view from open to closed
- Modified the client endpoints to call the `poll_calculation` function in order to pass the vote percentages in the item sent to the template for rendering when poll is closed

### Steps to test

1. Navigate to the Editor's interface.
2. Click "Add content here" and select Poll. Fill the poll form and submit. The poll will appear on the timeline.
3. Click "Live" to see the Live Preview. You will see the poll UI when open.
4. To view the poll UI when closed, wait for the poll time to elapse and it will change automatically to show the poll summary.


### Screenshots
#### When Poll is Open
![image](https://github.com/liveblog/liveblog/assets/14116447/6800c893-3d61-4cdf-828b-de3ccab94f9f)

#### When Poll is Closed
![image](https://github.com/liveblog/liveblog/assets/14116447/ea5a878b-bf77-4d07-b9df-5772574b53af)




Resolves: [LBSD-2767](https://sofab.atlassian.net/browse/LBSD-2767)
